### DISCUSSION TITLE
Including resources not directly connected

### DISCUSSION THREAD

**Discussion by SafaAlfulaij** [django-json-api/django-rest-framework-json-api#954] [General]
Sometimes I'd like to include a resource (notification for example) to a response without it being linked to any of the resource, or the linkage wil be hard.
I mentioned "notification" because it's the easiest, and I shouldn't treat it as `map` since it can diverge a lot.

**Comment by sliverc** [comment]
REST and JSON:API does not support something like this - if you have such use case you should consider using [GraphQL](https://graphql.org/).

This said if you have a simple case where one resource should always return the same non-related resources you could use [SerializerMethodResourceRelatedField](https://django-rest-framework-json-api.readthedocs.io/en/stable/usage.html#serializermethodresourcerelatedfield).
